hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -588,6 +598,20 @@ public void setBucketQuota(String volumeName, String bucketName,
         .setBucketName(bucketName)
         .setQuotaInBytes(quotaInBytes)
         .setQuotaInNamespace(quotaInNamespace);
+    // If the bucket is old, we need to remind the user on the client side
+    // that it is not recommended to enable quota.
+    OmBucketInfo omBucketInfo = ozoneManagerClient.getBucketInfo(
+        volumeName, bucketName);
+    if (omBucketInfo.getQuotaInNamespace() == OLD_QUOTA_DEFAULT) {
+      LOG.warn("Bucket {} is created before version 1.1.0, usedNamespace " +
+          "may be inaccurate and it is not recommended to enable quota.",
+          bucketName);
+    }
+    if (omBucketInfo.getUsedBytes() == OLD_QUOTA_DEFAULT) {
+      LOG.warn("Bucket {} is created before version 1.1.0, usedBytes " +
+          "may be inaccurate and it is not recommended to enable quota.",
+          bucketName);
+    }
```

Comment:
Can we pass this warnning message to client side?  Put it in log is not enough to raise the alert to user.

Reply:
Yes, the warning here is able to be displayed directly to the user client
![image](https://user-images.githubusercontent.com/13825159/104296852-1e7e8800-54fd-11eb-8507-51ad519e7586.png)

Reply:
@captainzmc ， we can see this output on console because LOG is enabled. When LOG is disabled on client side, we cannot see any message.  My suggestion is we return this message as the command response, and then output to console.